client/components/select-dropdown/item.jsx

```diff
@@ -3,18 +3,19 @@
 /**
  * External Dependencies
  */
-var React = require( 'react' ),
-	classNames = require( 'classnames' );
+const React = require( 'react' );
+const classNames = require( 'classnames' );
 
 /**
  * Internal dependencies
  */
-var Count = require( 'components/count' );
+const Count = require( 'components/count' );
 
-var SelectDropdownItem = React.createClass( {
+const SelectDropdownItem = React.createClass( {
 	propTypes: {
 		children: React.PropTypes.string.isRequired,
 		path: React.PropTypes.string,
+		isDropdownOpen: React.PropTypes.bool,
```

Comment:
No, for sure; however can I ask you if this has some particular React-specific benefit that I don't know about, or it will just improve the clarity of the component?